index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYh1E html>
+    <head>
+        <link rel="stylesheet" href="css/reset.css">

```

Comment:
Make sure that you have the following meta tag in your html so that your site will be responsive (if you open the console and click the mobile view, it will not show your mobile view with out the below code):

`<meta name="viewport" content="device-width, initial-scale=1">`